lib/twilio-interactor.rb

```diff
@@ -0,0 +1,104 @@
+class TwilioInteractor
```

Comment:
I think we need test coverage here. Did we have test coverage for those private methods anywhere else?

Reply:
We didn't directly and I didn't see anything in Cashbox. We can add them though.

Reply:
I've handled this!